WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListViewModel.kt

```diff
@@ -599,6 +654,10 @@ class PostListViewModel @Inject constructor(
                 localPostIdForPublishDialog = null
                 publishPost(it)
             }
+            CONFIRM_ON_CONFLICT_LOAD_REMOTE_POST_DIALOG_TAG -> localPostIdForConflictResolutionDialog?.let {
+                // here load version from remote
+                loadPostFromRemote(it)
```

Comment:
I think we should be nullifying the `localPostIdForConflictResolutionDialog` just before this similar to other dialogs. We should only hold onto that value while the dialog is shown and at this point, it'll be dismissed. We don't really need that information later either and if we did, it's better to pass it around than to hold onto a global property.

Reply:
good catch, addressed in 94948513da